esp/services/ws_ecl/ws_ecl_service.cpp

```diff
@@ -1629,6 +1629,12 @@ void CWsEclBinding::getWsEcl2XmlRequest(StringBuffer& soapmsg, IEspContext &cont
     }
 
     Owned<IPropertyTree> reqTree = createPTreeFromHttpParameters(wsinfo.queryname, parameters, true, false);
+    const char *statsToWorkunit = parameters->queryProp("save_stats");
+    if (statsToWorkunit && strieq(statsToWorkunit, "on"))
```

Comment:
You can use the strToBool(statsToWorkunit) method.  That supports on/true/1, etc.  Then it doesn't matter if it came from a form "on" or from a user adding a parameter which would likely be 1 or true.

Should we name these parameters exactly the same as we do for roxie?  It's ok to say no if it would cause confusion between form check boxes and actual parameters, but if they were common names it might make things cleaner.

We also need to make sure that if a soap request is sent to WsEcl, in the exact same format as the soap sent to roxie, it still works. (it should be a pass through when SOAP is used). It probably works fine but needs verification.

If these check boxes are checked, and they select the SOAP test form, it would be good if the parameters were set in the root of the soap message (just like the way they are set for the roxie message).